articles/user-guide/libraries/standard/data-structures.md

```diff
@@ -174,52 +176,59 @@ $$
+This is important to squeeze every last ounce of efficiency out of phase estimation algorithms. It allows you to choose precisely the experiment that provides the most information about $E$, whereas methods based on discrete queries must make do with compromising by choosing the best integer number of queries in the algorithm.
+
+As a concrete example of this, consider the problem of estimating not the rotation angle of a gate, but the procession frequency of a rotating quantum system.
+The unitary that describes such quantum dynamics is $U(t)=R_z(2\omega t)$, for evolution time $t$ and unknown frequency $\omega$.
+In this context, simulate $U(t)$ for any $t$ using a single $R_z$ gate and as such, you do not need to restrict yourself to only discrete queries to the unitary.
+Such a continuous model also has the property that frequencies greater than $2\pi$ can be learned from phase estimation processes that use continuous queries. This is because phase information that is otherwise masked by the branch-cuts of the logarithm function can be revealed from the results of experiments performed on non-commensurate values of $t$.
+Thus, for problems such as this, continuous query models for the phase estimation oracle are not only appropriate but are also preferable to the discrete query model.
+For this reason, Q# has functionality for both forms of queries and leaves it to the user to decide upon a phase estimation algorithm to fit their needs and the type of oracle that is available.
+
+> [!TIP]
+> You can further explore both of the oracle types described in this section in the Quantum Development Kit samples.
+> To learn more about continuous query oracles, see the [**PhaseEstimation** sample](https://github.com/microsoft/Quantum/tree/main/samples/characterization/phase-estimation).
+> To learn more about discrete query oracles, see the [**IsingPhaseEstimation** sample](https://github.com/microsoft/Quantum/tree/main/samples/simulation/ising/phase-estimation).
+
+## Dynamical generator modeling in Q\#
+
+Generators of time-evolution describe how states evolve through time. For example, the dynamics of a quantum state $\ket{\psi}$ is governed by the Schrödinger equation
 $$
 \begin{align}
     i\frac{d \ket{\psi(t)}}{d t} & = H \ket{\psi(t)},
 \end{align}
 $$
-with a Hermitian matrix $H$, known as the Hamiltonian, as the generator of motion. Given an initial state $\ket{\psi(0)}$ at time $t=0$, the formal solution to this equation at time $t$ may be, in principle, written
+with a Hermitian matrix $H$, known as the [Hamiltonian](xref:microsoft.quantum.chemistry.concepts.quantumdynamics), as the generator of motion. Given an initial state $\ket{\psi(0)}$ at time $t=0$, the formal solution to this equation at time $t$ may be, in principle, written
```

Comment:
This link does to the chemistry definition of a Hamiltonian, which is an important special case of the more general definition (a linear operator describing the energy of a system). If we need a link for this, I'd recommend making a new conceptual page for it, instead of communicating that chemistry is the only context in which Hamiltonian evolution occurs.